manifests/nexus.pp

```diff
@@ -0,0 +1,69 @@
+#
+# Download from Nexus using REST API
+# More info here: https://repository.sonatype.org/nexus-restlet1x-plugin/default/docs/path__artifact_maven_content.html
+#
+define archive::nexus (
+  $ensure       = present,
+  $checksum_type = 'md5',
+  $nexus_url    = undef,
+  $gav          = undef,
```

Comment:
It's gradle syntax for dependecy declaration:

http://mvnrepository.com/artifact/io.hawt/hawtio-web/1.4.49#gradle

Most developers says that it is more readable cause you find get all (important) package information in a single line.

The same is used here: https://github.com/cescoffier/puppet-nexus

And I'm not sure right now, but Nexus uses same GAV syntax when you're manually uploading an artifact.

What do you think? Declare each field (g, a, v)? Single field (gav)? Or support both?